nasl/nasl_packet_forgery.c

```diff
@@ -1196,11 +1374,43 @@ dump_tcp_packet (lex_ctxt *lexic)
       printf ("\tth_win   : %d\n", ntohs (tcp->th_win));
       printf ("\tth_sum   : 0x%x\n", ntohs (tcp->th_sum));
       printf ("\tth_urp   : %d\n", ntohs (tcp->th_urp));
-      printf ("\tData     : ");
-      c = (char *) ((char *) tcp + sizeof (struct tcphdr));
+
+      if (tcp->th_off > 5) // Options presents
+        {
+          char *options;
+          struct tcp_options *tcp_all_options;
+
+          options =
+            (char *) g_malloc0 (sizeof (uint8_t) * 4 * (tcp->th_off - 5));
+          memcpy (options, (char *) tcp + 20, (tcp->th_off - 5) * 4);
+
+          tcp_all_options = g_malloc0 (sizeof (struct tcp_options));
+          get_tcp_options (options, tcp_all_options);
+          if (tcp_all_options != NULL)
```

Comment:
The formatting of the output is wrong here. the value should come after the `:`. And would be nice to have them all aligned similar to the elements above. Same for ipv6 version.